(window.webpackJsonp=window.webpackJsonp||[]).push([[7],{"3XHS":function(e,t,a){"use strict";a.r(t);var r=a("q1tI"),n=a.n(r),l=(a("Wbzz"),a("vrFN")),i=a("j8Pw");t.default=function(){return n.a.createElement(i.a,null,n.a.createElement(l.a,{title:"About"}),n.a.createElement("div",{class:"row mb-4 px-3 pt-4 pb-2 tw-border-b-4 tw-border-gray-600"},n.a.createElement("div",{class:"col-12"},n.a.createElement("h4",null,"About NIAID Bioinformatics Resource Centers"),n.a.createElement("p",null,"The Bioinformatics Resource Centers (BRCs) for Infectious Diseases program was initiated in 2004 with the main objective of providing public access to computational platforms and analysis tools that enable collecting, archiving, updating, and integrating a variety of genomics and related research data relevant to infectious diseases, and pathogens and their interaction with hosts."," "),n.a.createElement("hr",null),n.a.createElement("h5",null,"Contact"),n.a.createElement("p",null,"For information on flu and other viral taxa, contact"," ",n.a.createElement("a",{href:"https://www.fludb.org/brc/problemReport.spg?decorator=influenza&category=science",target:"_blank"},"IRD"),","," ",n.a.createElement("a",{href:"https://www.viprbrc.org/brc/problemReport.spg?decorator=vipr&category=science",target:"_blank"},"ViPR")," ","(respectively)"),n.a.createElement("p",null,"For information on bacterial taxa, contact"," ",n.a.createElement("a",{href:"mailto:help@patricbrc.org"},"help@patricbrc.org")),n.a.createElement("p",null,"For information on protozoan & fungal taxa, or arthropod & mammalian hosts, contact"," ",n.a.createElement("a",{href:"mailto:help@veupathdb.org"},"help@veupathdb.org")),n.a.createElement("hr",null),n.a.createElement("h5",null,"Main Areas of Focus"),n.a.createElement("p",null,"The overarching goal of the BRC program is to ensure that high quality, large-scale datasets are readily Findable, Accessible, Interoperable, and Reusable (FAIR), expediting both discovery and translational research. We aim to:"),n.a.createElement("div",{className:"col-12"},n.a.createElement("ul",{style:{listStyle:"disc"}},n.a.createElement("li",null,"store, update, integrate, display, and facilitate interrogation of diverse data and associated metadata, including (but not limited to) information on: genome sequence and annotation, functional genomics information (transcriptomics, proteomics, etc.), population variation, genotype/phenotype association, metagenomics, epidemiology, surveillance, antimicrobial resistance, host responses & host-pathogen interactions, etc."),n.a.createElement("li",null,"make available analytical resources, bioinformatics tools, workspaces and services for data analysis, and provide bioinformatics training to ensure effective use of these resources"),n.a.createElement("li",null,"respond rapidly to new and emerging pandemic threats"))),n.a.createElement("hr",null),n.a.createElement("h5",null,"Targeted Pathogens"),n.a.createElement("p",null,"Each BRC specializes in a different group of pathogens. Pathogens examined by the BRCs include, but are not limited to, those in the NIAID list of emerging and re-emerging infectious diseases. The BRCs specialize in the following groups of pathogens:"),n.a.createElement("div",{className:"row"},n.a.createElement("div",{className:"col-md-12"},n.a.createElement("table",{className:"table"},n.a.createElement("tbody",null,n.a.createElement("tr",null,n.a.createElement("td",null,n.a.createElement("a",{href:"https://bv-brc.org/",target:"_blank"},n.a.createElement("strong",null,"Bacterial and Viral Bioinformatics Resource Center (BV-BRC)"))),n.a.createElement("td",null,n.a.createElement("strong",null,"Principal Investigators"),n.a.createElement("ul",null,n.a.createElement("li",null,"Rick Stevens, Univsity of Chicago"),n.a.createElement("li",null,"Richard Scheuermann, J. Craig Venter Institute")))),n.a.createElement("tr",null,n.a.createElement("td",{style:{paddingLeft:"40px"}},n.a.createElement("a",{href:"https://www.fludb.org/brc/home.spg?decorator=influenza",target:"_blank"},"IRD"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://www.viprbrc.org/brc/home.spg?decorator=vipr",target:"_blank"},"ViPR")),n.a.createElement("td",null,"Human viral families including influenza virus")),n.a.createElement("tr",null,n.a.createElement("td",{style:{paddingLeft:"40px"}},n.a.createElement("a",{href:"https://patricbrc.org",target:"_blank"},"PATRIC")),n.a.createElement("td",null,"Pathogenic bacteria and related species")),n.a.createElement("tr",null,n.a.createElement("td",null,n.a.createElement("a",{href:"https://veupathdb.org/veupathdb/app/",target:"_blank"},n.a.createElement("strong",null,"Eukaryotic Pathogen, Vector and Host Informatics Resource (VEuPathDB"),")")),n.a.createElement("td",null,n.a.createElement("strong",null,"Principal Investigators"),n.a.createElement("ul",null,n.a.createElement("li",null,"David S. Roos, University of Pennsylvania"),n.a.createElement("li",null,"Jessica C. Kissinger, University of Georgia"),n.a.createElement("li",null,"Mary Ann McDowell, University of Notre Dame")))),n.a.createElement("tr",null,n.a.createElement("td",{style:{paddingLeft:"40px"}},n.a.createElement("div",{style:{display:"inline-block"}},n.a.createElement("a",{href:"https://amoebadb.org/",target:"_blank"},"AmoebaDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://cryptodb.org/",target:"_blank"},"CryptoDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://fungidb.org/",target:"_blank"},"FungiDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://giardiadb.org/",target:"_blank"},"GiardiaDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://microsporidiadb.org/",target:"_blank"},"MicrosporidaDB"),n.a.createElement("br",null)),n.a.createElement("div",{style:{display:"inline-block",marginLeft:"30px"}},n.a.createElement("a",{href:"https://piroplasmadb.org/",target:"_blank"},"PiroplasmaDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://plasmodb.org/",target:"_blank"},"PlasmoDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://toxodb.org/",target:"_blank"},"ToxoDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://trichdb.org/",target:"_blank"},"TrichDB"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://tritrypdb.org/",target:"_blank"},"TriTrypDB"),n.a.createElement("br",null))),n.a.createElement("td",null,"Eukaryotic human pathogen species including fungi and protists")),n.a.createElement("tr",null,n.a.createElement("td",{style:{paddingLeft:"40px"}},n.a.createElement("a",{href:"https://vectorbase.org/",target:"_blank"},"VectorBase")),n.a.createElement("td",null,"Invertebrate vectors of human pathogens including insects, arachnids, and gastropods")),n.a.createElement("tr",null,n.a.createElement("td",{style:{paddingLeft:"40px"}},n.a.createElement("a",{href:"https://hostdb.org/",target:"_blank"},"HostDB")),n.a.createElement("td",null,"Vertebrate hosts")),n.a.createElement("tr",null,n.a.createElement("td",{style:{paddingLeft:"40px"}},n.a.createElement("a",{href:"https://orthomcl.org/",target:"_blank"},"OrthoMCL")),n.a.createElement("td",null,"Ortholog groups of protein sequences")))),n.a.createElement("hr",null),n.a.createElement("h5",null,"Website Usage"),n.a.createElement("a",{href:"https://status.patricbrc.org/webstats/cgi-bin/awstats.pl?config=All_BVBRC_Combined",target:"_blank"},"BV-BRC Stats"),n.a.createElement("br",null),n.a.createElement("a",{href:"https://veupathdb.org/awstats/awstats.pl?config=All_EBRC_Combined",target:"_blank"},"VEuPathDB Stats"))))))}},j8Pw:function(e,t,a){"use strict";var r=a("q1tI"),n=a.n(r),l=(a("Wbzz"),a("Iu+6"),a("rL/g"),a("MHGr"),a("rMck"),a("+YBn"),a("rY4l")),i=(a("uP3y"),a("VsrT"),a("JwsL"));t.a=function(e){var t=e.children;return n.a.createElement(n.a.Fragment,null,n.a.createElement(l.a,{siteTitle:"Title"}),n.a.createElement("div",null,n.a.createElement("main",{role:"main"},n.a.createElement("div",{class:"container-fluid max-1600"},n.a.createElement("div",{class:"row"},n.a.createElement("div",{class:"col-md-12 px-5 pt-3"},t))))),n.a.createElement(i.a,null))}}}]);
//# sourceMappingURL=component---src-pages-about-js-71d908b5e58a5b432d89.js.map